include/boost/wintls/detail/sspi_handshake.hpp

```diff
@@ -234,18 +240,19 @@ class sspi_handshake {
       }
 
       case SEC_I_INCOMPLETE_CREDENTIALS:
-        BOOST_ASSERT_MSG(false, "client authentication not implemented");
+        WINTLS_ASSERT_MSG(false, "client authentication not implemented");
 
       case SEC_I_RENEGOTIATE:
-        BOOST_ASSERT_MSG(false, "renegotiation not implemented");
+        WINTLS_ASSERT_MSG(false, "renegotiation not implemented");
 
       default:
         return state::error;
     }
   }
 
   void size_written(std::size_t size) {
-    BOOST_VERIFY(size == out_buffer_.size());
+    (void)(size);
+    assert(size == out_buffer_.size());
```

Comment:
If the `BOOST_VERIFY_MSG` macro is reimplemented, might as well reimplement both of them.